labelbox/schema/model_run.py

```diff
@@ -32,6 +32,7 @@ class ModelRun(DbObject):
     created_at = Field.DateTime("created_at")
     created_by_id = Field.String("created_by_id", "createdBy")
     model_id = Field.String("model_id")
+    training_metadata = Field.Json("training_metadata")
```

Comment:
FYI, adding this field here means that graphql query will now expect `trainingMetadata` from the resolver response. This may break until the backend is updated to include this field

Reply:
Yeah, the resolver has already been updated with this optional field!